examples/gno.land/r/demo/realm_banker/realm_banke_test.gno

```diff
@@ -0,0 +1,69 @@
+package realm_banker
```

Comment:
typo in filename?

s/realm_banke_test.gno/realm_banker_test.gno

Reply:
https://github.com/gnolang/gno/pull/875/commits/58ddfd14bc6cca19696b3eb042f2ae9cfe34bcd1